service/src/main/java/org/cbioportal/service/impl/GeneServiceImpl.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2016 Memorial Sloan Kettering Cancer Center.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS
+ * FOR A PARTICULAR PURPOSE. The software and documentation provided hereunder
+ * is on an "as is" basis, and Memorial Sloan-Kettering Cancer Center has no
+ * obligations to provide maintenance, support, updates, enhancements or
+ * modifications. In no event shall Memorial Sloan-Kettering Cancer Center be
+ * liable to any party for direct, indirect, special, incidental or
+ * consequential damages, including lost profits, arising out of the use of this
+ * software and its documentation, even if Memorial Sloan-Kettering Cancer
+ * Center has been advised of the possibility of such damage.
+ */
+
+/*
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+package org.cbioportal.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.cbioportal.model.Gene;
+import org.cbioportal.persistence.GeneRepository;
+import org.cbioportal.service.GeneService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/**
+ *
+ * @author jiaojiao
+ */
+@Service
+public class GeneServiceImpl implements GeneService{
+    @Autowired
+    private GeneRepository geneRepository;
+    
+    @Override
+    public List<Gene> getGeneListByHugoSymbols(List<String> hugo_gene_symbol){
+        if(hugo_gene_symbol == null || hugo_gene_symbol.size() == 0)
```

Comment:
Style comments: add a space after keywords (like "if") .. and keep the opening  "{" on the same line as the if statement. add spaces around "else", and add a space between ")" and "{" in the function declaration line.